Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM eclipse-temurin:8-jdk-alpine
```

Comment:
this has already been updated for java 17. any chance you may have made your changes to outdated git code?